lib/data_update_scripts/20211104200856_update_each_articles_privileged_user_point_count.rb

```diff
@@ -0,0 +1,17 @@
+module DataUpdateScripts
+  class UpdateEachArticlesPrivilegedUserPointCount
+    def run
+      # This code is "idempotent" in that it's calculating and caching a value stored elsewhere.
+      Article.find_in_batches do |articles|
+        articles.each do |article|
+          # Given that we have a default of 0, don't update if we
+          # don't have any reactions.
+          next unless article.reactions.privileged_category.exist?
+
+          article.update_column(:privileged_users_reaction_points_sum,
+                                article.reactions.privileged_category.sum(:points))
+        end
+      end
```

Comment:
Nitpick, especially considering this code is not expected to be maintained long-term: [`AR::Base#find_each`](https://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each) does this without nested blocks.

```suggestion
      Article.find_each do |article|
        # Given that we have a default of 0, don't update if we
        # don't have any reactions.
        next unless article.reactions.privileged_category.exist?

         update_column(:privileged_users_reaction_points_sum,
                       article.reactions.privileged_category.sum(:points))
      end
```